routes/studentRoutes.js

```diff
@@ -1,241 +1,220 @@
 const router = require('express').Router();
 const errorCodes = require('../helpers/errorCodes');
-const { markAsCompleted } = require('../helpers/answerExercises');
+const { markAsCompleted } = require('../helpers/completing');
 const requireLogin = require('../middlewares/requireLogin');
 const mongoose = require('mongoose');
-const { encrypt, compare } = require('../helpers/password');
 const { findTop100Students } = require('../helpers/leaderboard');
-const { updateStudentLevel, giveExtraPointsForSection } = require('../helpers/pointSystem');
-
-const { ExerciseModel } = require('../models/Exercises');
+const { addIncompleteCourse } = require('../helpers/completing');
 const { StudentModel } = require('../models/Students');
-const { SectionModel } = require('../models/Sections');
 const { CourseModel } = require('../models/Courses');
 
-router.patch('/:id', requireLogin, async (req, res) => {
-  if (!mongoose.Types.ObjectId.isValid(req.params.id)) {
-    return res.status(400).send({ error: errorCodes['E0014'] });
-  }
-
-  const id = mongoose.Types.ObjectId(req.params.id);
-
-  const { points, level } = req.body;
-
-  let student = await StudentModel.findOne({ baseUser: id });
-
-  let updatedUser;
+router.get('/:id/info', async (req, res) => {
```

Comment:
the 'info' should be removed from this route

Reply:
@MaizeHH why?

Reply:
the api guideline does it like that. we'll have a look and see if it is worth fixing at this point

Reply:
This was also mentioned to the group who made this. If you change it it should be changed to '/info/:id'
If you change it, it should also be changed on the app